app/funcs/csv_reader.py

```
@@ -5,7 +5,7 @@
 from typing import Callable, Dict, Generator
 import datetime
 from dataclasses import dataclass
-from logger import Logger
+from app.logger import Logger
 from rich.progress import Progress
 import polars as pl
 from rich.progress import Progress
```

Ruff (0.14.7)

Redefinition of unused `Progress` from line 9

Remove definition: `Progress`

(F811)